tensorflow/g3doc/get_started/os_setup.md

```diff
@@ -635,7 +635,7 @@ copy of Python.
 
 On El Capitan, "six" is a special package that can't be modified, and this
 error is reported when "pip install" tried to modify this package. To fix use
-"ignore_installed" flag, ie
+"ignore_installed" flag e.g.,
```

Comment:
++

This change was not meant to be included. I'll remove and update the pull.